test/distributed/test_c10d_nccl.py

```diff
@@ -4064,6 +4062,8 @@ def test_individual_send_recv(self, op_sizes, timing_enabled):
             input_sizes = op_sizes[seq % ops_per_repeat]
             profiling_name = "nccl:recv 0<-1" if self.rank == 0 else "nccl:send 1->0"
             self.assertEqual(t["entries"][seq]["profiling_name"], profiling_name)
+            # we don't increment collective_seq_id for p2p ops.
```

Comment:
hmm. one more possible wrinkle, since i am clearly trying to be annoying here, is we actually do use the 'collective' communicator when doing coalesced p2ps.  In other words, i think it might have been intentional to bump the collective seq for coalp2p.  Someone check me on this?

ref: we get a special nccl stream and nccl comm for p2p ops, but iirc we use the normal stream/comm for coalp2p same as we would use for collectives.

given this, what makes the most sense from a logging pov?  clearly it will confuse users if we increment collective seq when we do p2p.  otoh, if we are trying to do trace analysis to debug a hang, it might be important to note that this coalp2p happened sequentially after a collective on the same ncclcomm.